.eslintrc.json

```diff
@@ -1,68 +1,56 @@
 {
-    "env": {
-        "browser": true,
-        "es2022": true,
-        "node": true,
-        "jest": true
-    },
-    "extends": [
-        "plugin:jsdoc/recommended",
-        "eslint:recommended",
-        "plugin:@typescript-eslint/eslint-recommended",
-        "plugin:@typescript-eslint/recommended",
-        "next/core-web-vitals",
-        "next"
+  "env": {
+    "browser": true,
+    "es2022": true,
+    "node": true,
+    "jest": true
+  },
+  "extends": [
+    // "plugin:jsdoc/recommended",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "next/core-web-vitals",
+    "next"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["header", "jsdoc", "@typescript-eslint", "check-file"],
+  "rules": {
+    "header/header": [
+      "warn",
+      "block",
+      " This Source Code Form is subject to the terms of the Mozilla Public\n * License, v. 2.0. If a copy of the MPL was not distributed with this\n * file, You can obtain one at http://mozilla.org/MPL/2.0/. ",
+      2
     ],
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "ecmaVersion": "latest",
-        "sourceType": "module"
-    },
-    "plugins": [
-        "header",
-        "jsdoc",
-        "@typescript-eslint",
-        "check-file"
+    "jsdoc/require-jsdoc": "off",
+    "jsdoc/require-param-description": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    // Unused vars that start with an understore are allowed to be unused:
+    "@typescript-eslint/no-unused-vars": [
+      "off",
```

Comment:
Disabling this temporarily.

Reply:
I restored it and removed the offending variables (or added a comment why they're fine to have) in 6419bda6b.